lib/linguist/languages.yml

```diff
@@ -3078,7 +3078,12 @@ Nim:
   color: "#37775b"
   extensions:
   - ".nim"
+  - ".nim.cfg"
```

Comment:
Could you also add a sample file for this file extension?

Reply:
done